.yalc/@lidofinance/next-api-wrapper/README.md

```diff
@@ -0,0 +1,100 @@
+# @lidofinance/next-api-wrapper
```

Comment:
You forgot to add .yalc to .gitignore

Reply:
Not forgotten. I will delete `.yalc` after packages will be published